src/cli/generate-config.js

```diff
@@ -83,7 +85,9 @@ export default function generateConfig(config) {
   delete config.devServer;
 
   // Set publicPath
-  config.output.publicPath = `http://localhost:8080/${huron.root}`;
+  if (isHot) {
```

Comment:
I think we should probably use `if (!program.production)` for this instead of relying on the npm lifecycle event